src/pages/School/Admin/SchoolEdit.tsx

```diff
@@ -103,24 +127,80 @@ const SchoolEdit: FC = () => {
       <Button theme="darkBlue" onClick={() => setType('heart')} text="Donate" />
       <Button theme="darkBlue" onClick={() => setType('plus')} text="Excess" />
       <div className={`${styles.banner} ${styles[type]}`}>
-        <h2>{banner}</h2>
+        <h2>{content.banner}</h2>
```

Comment:
Any way of destructing content above so you don't have to notate throughout the jsx?

Reply:
I could destructure it however I need these notated values to update when the stateful variable content changes so I don't know if it makes too much difference either way?

Reply:
Yeh, it's just to improve readability/reduce repetition, but I'll resolve